stuff/pipelines/database/contest.py

```diff
@@ -42,14 +42,13 @@ def get_contest_contestants(
             finish_time BETWEEN %s AND %s;
         ''', (date_lower_limit, date_upper_limit)
     )
-    data: List[ContestCertificate] = list()
+    data: List[Dict[str, str]] = []
     for row in cur:
-        data.append(
-            ContestCertificate(
-                certificate_cutoff=row['certificate_cutoff'],
-                alias=row['alias'],
-                scoreboard_url=row['scoreboard_url'],
-                contest_id=row['contest_id'],
-            )
+        contest = ContestCertificate(
+            certificate_cutoff=row['certificate_cutoff'],
+            alias=row['alias'],
+            scoreboard_url=row['scoreboard_url'],
+            contest_id=row['contest_id'],
         )
+        data.append(contest._asdict())
```

Comment:
```suggestion
        data.append(contest)
```

hay que intentar mantener todo typesafe hasta el momento en el que se necesite tener no-typesafe.